static/app/components/events/interfaces/spans/header.tsx

```diff
@@ -614,6 +614,7 @@ const TimeAxis = styled('div')`
   color: ${p => p.theme.gray300};
   font-size: 10px;
   font-weight: 500;
+  font-variant-numeric: tabular-nums;
```

Comment:
@vuluongj20 Can you apply this to the `<TickText/>` component in this same file (just below)?

Reply:
And `<DurationGuideBox/>`

Reply:
Hmm both of these are nested inside `<TimeAxis/>`, and should inherit the style from it:

<img width="487" alt="Screen Shot 2021-09-27 at 3 45 18 PM" src="https://user-images.githubusercontent.com/44172267/134995344-ed4f14fe-4805-4ef8-8894-a1f7a6b6df6c.png">

Did you just want me to be more specific (i.e. apply `tabular-nums` to `<TickText/>` and `<DurationGuideBox/>` instead of `<TimeAxis/>` as the shared parents)?

Reply:
@vuluongj20 ah i see. looks good to me!